src/crypto/shelley/transactionUtils.js

```diff
@@ -103,7 +103,7 @@ export const createUnsignedTx = async (
         [], // no certificates
         [], // no withdrawals
         false, // do not allow no outputs
```

Comment:
you don't have to fix it  but I just wanted to point out that this line is....ummm..... what the what?!?